resources/site/how-to/workbench-application-platform/how-to-issue-custom-intent.md

```diff
@@ -0,0 +1,47 @@
+![SCION Workbench](/resources/site/logo/scion-workbench-banner.png)
+
+[Overview][menu-overview] | [Workbench][menu-workbench] | [Workbench&nbsp;Application&nbsp;Platform][menu-workbench-application-platform] | [Contributing][menu-contributing] | [Changelog][menu-changelog] | [Sponsoring][menu-sponsoring] | [Links][menu-links]
+|---|---|---|---|---|---|---|
+
+## Issuing a custom intent to the host application
+
+The workbench application platform provides an API to register [custom intent handlers](how-to-install-a-programmatic-intent-handler.md) in the host application. A child application can then use the `IntentService` to issue a generic intent to the platform. As those intents are generic, a client has to know what qualifier and payload he has to provide and what the response will be.
+
+The `IntentService` offers two methods:
+
+|method|description|
+|-|-|
+|issueIntent$|This method issues an intent to the platform and returns a stream of replies. The response type has to be provided by the caller (see the example below)|
+|issueIntent|This method issues an intent without expecting a reply (fire & forget)|
+
+### Example with stream of replies
+```typescript
+constructor(private _intentService: IntentService) { } 
+
+public onIssueIntent(): void {
+   this.result$ = this._intentService.issueIntent$<string>('my-type', {'entity': 'example'}, 'my-payload')
+   .pipe(
+     takeUntil(this._$destroy) // the client has to cleanup his subscription onDestroy
+   )
+}
+```
+
+### Example with exactly one response
+```typescript
+constructor(private _intentService: IntentService) { } 
+
+public onIssueIntent(): void {
+   this.result$ = this._intentService.issueIntent$<string>('my-type', {'entity': 'example'}, 'my-payload')
+    .pipe(
+      first() // as the client knows that only one reply will be sent, he can terminate the subscription immediately
+    )
+}
+```
+
```

Comment:
Please mention how to get the service for non-Angular applications.
`> For non Angular applications, get the intent service via Platform.getService(IntentService).`